### DISCUSSION TITLE
New default Volar 2 configuration

### DISCUSSION THREAD

**Discussion by DanielViberg** [yegappan/lsp#485] [Ideas]
Since Volar 2.0.7 it is neccessary to set the hybridMode = false in order for it to not require tsserver running in parallel.
@yegappan Could you please update the wiki?
`npm -g i @vue/language-server@latest`
```json
{
      "name": "vue-ls",
      "filetype": ["vue"],
      "path": "vue-language-server",
      "args": ["--stdio"],
      "initializationOptions": {
          "typescript": {
              "tsdk": ".../node/v20.11.0/lib/node_modules/typescript/lib"
          },
          "vue": {
            "hybridMode": false
          }
      }
}

```

**Comment by yegappan** [comment]
Thanks. I have updated the wiki with this information.  Can you please review the changes?

**Reply by DanielViberg** [reply]
I should note that vue-ls = volar 2, so it should link to https://github.com/vuejs/language-tools and maybe note that if using version >= 2.0.7 the user need to add the hybridMode setting to the init options.

**Reply by yegappan** [reply]
I have updated the wiki to reflect that it is for Volar 2.

**Reply by DanielViberg** [reply]
Looks good!
